### PR TITLE
PL-1588: add realms

### PR DESCRIPTION
**Pull request by VKCR**

### CODE REVIEW

File: src/http/HttpClient.js [hunk 1]
```diff
@@ -68,15 +68,20 @@ class HttpClient {
...
      *
      * @param {Object} alias - alias to lookup
+     * @param {string} realm - realm of the alias
      * @return {Object} response - response to the API call
      */
-    resolveAlias(alias) {
+    resolveAlias(alias, realm) {
+        if (realm === undefined) {
+            realm = "";
+        }
```
Comment (zixunjian) [inline src/http/HttpClient.js]: can be simplified by doing `resolveAlias(alias, realm='')`, also our style standard is single quotes in JS.

Comment (VKCR) [reply src/http/HttpClient.js]: fixed

**Comment by colin-token** [comment]
Comments should indicate that realm is optional

**Comment by colin-token** [comment]
Also we need sdk-js-tests for this as well

File: src/Util.js [hunk 2]
```diff
@@ -39,6 +39,15 @@ class Util {
         };
     }
 
+    /**
+     * Returns the token realm.
+     *
+     * @return {string} token realm
+     */
+    static tokenRealm() {
+        return "token";
```
[the remark this answers is not in the document]
Comment (VKCR) [reply src/Util.js]: done

**Comment by VKCR** [comment]
added optional to realm comments. Will write tests in the future.

**Pull request merged**